Makefile

```diff
@@ -42,7 +42,6 @@ chart_setup:
 	mkdir -p ${CHART_PATH}/.packaged
 
 chart_package:
-	echo "appVersion: ${IMAGE_VERSION}" >> ${CHART_PATH}/${CHART_NAME}/Chart.yaml
```

Comment:
We still need to generate the appversion from the IMAGE_VERSION otherwise when the unstable builds are done it'll be using the wrong image.

Reply:
Fixed.